src/Cairo.jl

```diff
@@ -100,8 +100,12 @@ end
 
 function read_from_stream_callback(s::IO, buf::Ptr{UInt8}, len::UInt32)
     # wrap the provided buf into a julia Array
-    b1 = unsafe_wrap(Array{UInt8,1},buf,len)
 
+    if VERSION < v"0.5.0-rc"
+        b1 = pointer_to_array(buf,len); #pre-v0.5
+    else
+        b1 = unsafe_wrap(Array{UInt8,1},buf,len);
```

Comment:
I've seen this after committing ... but i had strange testing results (OSX succ, linux fail) with 0.5, so i need another run anyway.